src/EnergyPlus/DataSurfaces.hh

```diff
@@ -178,6 +178,15 @@ namespace DataSurfaces {
         Num
     };
 
+    enum RefAirTemp // Parameters to indicate reference air temperatures for inside surface temperature calculations
+    {
+        Invalid = -1,
+        ZoneMeanAirTemp,   // mean air temperature of the zone => MAT
+        AdjacentAirTemp,   // air temperature adjacent ot surface => TempEffBulkAir
```

Comment:
@dareumnam while your making changes, "adjacent ot surface" -> "adjacent to surface"